src/app/services/chart.service.ts

```diff
@@ -13,8 +13,23 @@ import * as _ from 'lodash';
 @Injectable()
 export class ChartService {
 
+    private multiChartScrubberInfo = new Subject();
+    private multiChartScrubberHover = new Subject<Boolean>();
+
+    multiChartScrubberInfo$ = this.multiChartScrubberInfo.asObservable();
```

Comment:
Looks like it's a pattern [from Cycle.js](https://cycle.js.org/basic-examples.html#what-does-the-suffixed-dollar-sign-mean), I've seen it used in a couple places but it seems to stem from them.